src/App.tsx

```diff
@@ -1,61 +1,218 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import React, { useState } from 'react';
+
+export type User = (typeof usersFromServer)[0];
+
+export interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  createdAt: Date;
+}
+
+export interface UserTodos {
+  userId: number;
+  userName: string;
+  userEmail: string;
+  userLogin: string;
+  todoPosts: Todo[];
+}
+
+const usersTodos = usersFromServer
+  .map(user => {
+    const userTodo = todosFromServer.reduce((todosAcc: Todo[], todo) => {
+      if (user.id === todo.userId) {
+        todosAcc.push({ ...todo, ...{ createdAt: new Date() } });
+
+        return todosAcc;
+      }
+
+      return todosAcc;
+    }, []);
```

Comment:
You don't have to make your app more complicated

Reply:
Here I created a new data structure and filtered users only having any todo posts